websites/views_test.py

```diff
@@ -650,14 +650,14 @@ def test_websites_content_list(drf_client, filter_type, permission_groups):
         ),
         {"type": filter_type},
     )
-    assert resp.data["count"] == (num_pages if filter_type else num_pages + 1)
+    assert len(resp.data) == (num_pages if filter_type else num_pages + 1)
```

Comment:
Minor nitpick: This isn't really a number of pages, maybe change the variable name to something like `num_results`?

Reply:
Ah, makes sense